routers/repo/attachment.go

```diff
@@ -92,6 +92,7 @@ func DeleteAttachment(ctx *context.Context) {
 
 // GetAttachment serve attachements
 func GetAttachment(ctx *context.Context) {
+	log.Info("GetAttachment(%s)", ctx.Params(":uuid"))
```

Comment:
Do we really want to log these on `Info` level?
I thought `access.log` was the place for this information

Reply:
oh crap sorry these are from another problem. Sorry.